app/invocation/invocation_action_card.tsx

```diff
@@ -475,6 +492,7 @@ export default class InvocationActionCardComponent extends React.Component<Props
   }
 
   fetchProfile(executionId: string) {
+    this.setState({ profileLoading: true });
```

Comment:
probably should attach a `.finally` handler to the RPC below which calls `this.setState({ profileLoading: false })`